src/pages/filter/ui/panels/function-panel/components/gene-region/gene-region.tsx

```diff
@@ -23,10 +23,10 @@ export const GeneRegion = observer(() => {
   }
 
   useEffect(() => {
-    const params: string = `{"locus":"${locusValue}"}`
+    const params: string = selectedFilterValue
```

Comment:
I think `params` variable is not necessary, you can just inline `selectedFilterValue` as function argument